docs/source/sctool/partials/sctool_restore.yaml

```diff
@@ -0,0 +1,158 @@
+- name: batch-size
+  default_value: "2"
+  usage: |
+    Number of SSTables per shard to process in one request by one node.
+    Increasing batch size increases job granularity.
+- name: cluster
+  shorthand: c
+  usage: |
+    The target cluster `name or ID` (envvar SCYLLA_MANAGER_CLUSTER).
+- name: cron
+  usage: |
+    Task schedule as a cron `expression`.
+    It supports the extended syntax including @monthly, @weekly, @daily, @midnight, @hourly, @every X[h|m|s].
+- name: dry-run
+  default_value: "false"
+  usage: |
+    Validates and displays restore information without actually scheduling the restore.
+    This allows you to display what will happen should the restore run with the parameters you set.
+- name: enabled
+  default_value: "true"
+  usage: |
+    Not enabled tasks are not executed and hidden from task list.
+- name: help
+  shorthand: h
+  default_value: "false"
+  usage: help for restore
+- name: interval
+  shorthand: i
+  usage: |
+    --interval is deprecated, please use `--cron` instead
+    Time after which a successfully completed task would be run again, supported units are:
+
+    'd' - days
+    'h' - hours
+    'm' - minutes
+    's' - seconds
+    'ms' - milliseconds
+
+    The task run date is aligned with '--start date' value.
+    For example, if you select '--interval 7d' task would run weekly at the '--start-date' time.
+- name: keyspace
+  shorthand: K
+  default_value: '[]'
+  usage: |+
+    A list of `glob` patterns separated by a comma used to include or exclude tables.
+    The patterns match keyspaces and tables, separate the keyspace name from the table name with a dot e.g. 'keyspace,!keyspace.table_prefix_*'.
+    The following syntax for glob patterns is supported:
+
+    '*' - matches any number of any characters including none
+    '?' - matches any single character
+    '[abc]' - matches one character given in the bracket
+    '[a-z]' - matches one character from the range given in the bracket
+
+    Patterns are evaluated from left to right.
+    If a pattern starts with '!' it unselects items that were selected by previous patterns i.e. 'a?,!aa' selects *ab* but not *aa*.
+
+- name: location
+  shorthand: L
+  default_value: '[]'
+  usage: |
+    A list of backup locations separated by a comma, specifies places where restored backup is stored, the format is `[<dc>:]<provider>:<bucket>`.
+    The '<dc>' parameter is optional it allows to specif datacenter which nodes will be used to restore data
+    from this location in a multi-dc setting, it must match Scylla nodes datacenter.
+    By default, all live nodes are used to restore data from specified locations.
+    Note that specifying datacenters closest to backup locations might reduce download time of restored data.
+    The supported storage '<provider>'s are 'azure', 'gcs', 's3'.
+    The 'name' parameter is a bucket name, it must be an alphanumeric string and **may contain a dash and or a dot, but other characters are forbidden**.
```

Comment:
This is not clear to me. Where is the `name` parameter? Does this belong here?
On the other hand, there is a `bucket` parameter, which is not described, and it's not clear what it refers to.
This line should be rewritten (but I'm not sure how).